sessionctx/variable/sysvar_test.go

```diff
@@ -518,7 +518,8 @@ func (*testSysVarSuite) TestInstanceScopedVars(c *C) {
 
 	val, err = GetSessionOrGlobalSystemVar(vars, TiDBRecordPlanInSlowLog)
 	c.Assert(err, IsNil)
-	c.Assert(val, Equals, strconv.FormatUint(uint64(atomic.LoadUint32(&config.GetGlobalConfig().Log.RecordPlanInSlowLog)), 10))
+	enabled := atomic.LoadUint32(&config.GetGlobalConfig().Log.RecordPlanInSlowLog) == 1
+	c.Assert(val, Equals, BoolToOnOff(enabled))
```

Comment:
This is a boolean typed value, but it was previously returning "1" instead of "ON/OFF". Because this code cahnge now calls validation on getters, it needed to be fixed.